<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScheduleEntryDialog</class>
 <widget class="QDialog" name="ScheduleEntryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add scheduled speed limit</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="label1">
       <property name="text">
        <string>Download</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label2">
       <property name="text">
        <string>Upload</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label3">
       <property name="text">
        <string>Pause?</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label4">
       <property name="text">
        <string>Speed limit</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="downloadSpinBox">
       <property name="specialValueText">
        <string>∞</string>
       </property>
       <property name="suffix">
        <string> KiB/s</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="uploadSpinBox">
       <property name="specialValueText">
        <string>∞</string>
       </property>
       <property name="suffix">
        <string> KiB/s</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QCheckBox" name="pauseCheckBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label5">
       <property name="text">
        <string>In between</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTimeEdit" name="timeEditFrom">
       <property name="wrapping">
        <bool>true</bool>
       </property>
       <property name="time">
        <time>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QTimeEdit" name="timeEditTo">
       <property name="wrapping">
        <bool>true</bool>
       </property>
       <property name="time">
        <time>
         <hour>23</hour>
         <minute>59</minute>
         <second>59</second>
        </time>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="labelInvalidStart">
       <property name="styleSheet">
        <string notr="true">color: red</string>
       </property>
       <property name="text">
        <string>Invalid</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="labelInvalidEnd">
       <property name="styleSheet">
         <string notr="true">color: red</string>
       </property>
       <property name="text">
        <string>Invalid</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ScheduleEntryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ScheduleEntryDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pauseCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>downloadSpinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pauseCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>uploadSpinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
